content/updates/connectivity/2-Enhanced-connection-monitoring.md

```diff
@@ -0,0 +1,46 @@
+# Enhanced connection monitoring
+
+## Monitor errors on your connections 
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+For each of your connections, a new `Monitoring` page now lists the last integration errors that may have occurred.
+
+From this page, you can now easily track and fix any data error on product creation, update, or deletion for each of your connections. 
+
+![Error monitoring page](../img/error-monitoring-page-header.png)
+
+Each warning message shows:
+- a small paragraph that explains what went wrong during the connection's synchronization, 
+- internal links to help you check the data in your PIM,
+- external links to directly access useful information in our help center.
+
+![Enriched error message](../img/fix-errors-on-your-connections.png)
+
+We also enhanced the Connection dashboard with additional information to ease error monitoring and allows you to see at a glance how your source connections are performing.  
+
+To ease error monitoring and allow you to see at a glance how your source connections are performing, the Connection dashboard now displays: 
```

Comment:
lines 20 and 22 are the same, aren't they?

Reply:
You're right. I'll reformulate.

Reply:
Done.